app/jobs/dqt/sync_states_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Dqt
+  class SyncStatesJob < ApplicationJob
+    queue_as :default
+    retry_on Client::HttpError
+
+    BATCH_SIZE = 100
+    INTERVAL = 30.seconds
+
+    def perform
+      return unless FeatureService.enabled?(:integrate_with_dqt)
+
+      # DQT have a validation on length of TRN
+      trainees = Trainee.trn_received
+                        .imported_from_hesa
```

Comment:
Did we set the trainees that we got from DTTP but that were originally from HESA as `.imported_from_hesa`? I think we did. Just checkin'.

Reply:
we did! `imported_from_hesa` actually just means 'does it have a hesa_id' 🙃